http-server/src/test/resources/datasets/integration.yml

```diff
@@ -0,0 +1,13 @@
+error_report_history:
```

Comment:
This is the dataset for the client-server integration test. Notably needed so we have a specific API key and can better exercise endpoints by passing a valid key.